it/lessons/basics/enum.md

```diff
@@ -9,14 +9,36 @@ Un insieme di algoritmi per enumerare le collezioni.
 
 ## Enum
 
-Il modulo `Enum` contiene oltre un centinaio di funzioni per lavorare con le collezioni di cui abbiamo parlato nella scorsa lezione.
+Il modulo `Enum` contiene oltre un 70 funzioni per lavorare con le collezioni di cui abbiamo parlato nella scorsa lezione.
+Tuttle le collezione di cui abbiamo imparato nella lezione precendente, eccetto le tuple. sono enumerazioni.
 
-Questa lezione tratterà solo una parte delle funzioni disponibili, per conoscere tutte le funzioni visita la documentazione ufficiale del modulo [`Enum`](https://hexdocs.pm/elixir/Enum.html); per enumerare gli elementi a richiesta (_lazy enumeration_) puoi usare il modulo [`Stream`](https://hexdocs.pm/elixir/Stream.html).
+Questa lezione tratterà solo una parte delle funzioni disponibili, ma possiamo esaminarle da soli
+Facciamo un piccolo esperimento in IEx.
+
+```elixir
+iex> Enum.__info__(:functions) |> Enum.each(fn({function, arity}) ->
+...>   IO.puts "#{function}/#{arity}"
+...> end)
+all?/1
+all?/2
+any?/1
+any?/2
+at/2
+at/3
+...
+```
+
+Usando questo, è especialmente chiaro che abbiamo una grande quantità di funzionalità e questo è per una specifica ragione.
```

Comment:
`especialmente` looks like spanish, I'd suggest to use one of the following:

* `particolarmente`
* omit it at all, it makes sense anyway

Reply:
fixed in efe71e5a7017e4a5602d069b368c9c56d9880e4b